runtime/coverage.go

```diff
@@ -539,3 +552,25 @@ func (r *CoverageReport) MarshalLCOV() ([]byte, error) {
 
 	return buf.Bytes(), nil
 }
+
+// Given a common.Location, returns its mapped source, if any.
+// Defaults to the location's ID().
+func (r *CoverageReport) locationSource(location common.Location) string {
```

Comment:
maybe `sourcePathForLocation` is more intuitive?

Reply:
Good point :+1: 
Updated in https://github.com/onflow/cadence/pull/2859/commits/4871d50d108d715a582746143ff09d72e78c525b